lisa/transformers/kernel_source_installer.py

```diff
@@ -39,6 +38,12 @@ class LocalLocationSchema(BaseLocationSchema):
             required=True,
         ),
     )
+    config_path: str = field(
```

Comment:
The name `config_path` is not clear. It needs to be more specific.

Reply:
This is .config path which is needed while building the kernel so i name it like config_path

Reply:
how about "kernel_config_file"?

Reply:
sure Chi, will make that change.